TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/controller/BotDirection.java

```diff
@@ -0,0 +1,8 @@
+package com.novoda.tpbot.controller;
+
+public enum BotDirection {
```

Comment:
I would keep this as a generic `Direction`, so that if we want to reuse the DirectionPad component in other project, we are not leaking the bot concepts